Dockerfile

```
@@ -20,7 +20,7 @@ ENV KLAYTN_DISABLE_SYMBOL=$KLAYTN_DISABLE_SYMBOL
 ADD . $SRC_DIR
```

During the `make all` execution, `git status` is commanded. However, a container doesn't have `.git` file. 
```suggestion
RUN git init
ADD . $SRC_DIR
```